datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp.py

```diff
@@ -0,0 +1,102 @@
+# (C) Datadog, Inc. 2020
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import os
+
+import click
+import yaml
+
+from ..console import CONTEXT_SETTINGS
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, short_help='SNMP utilities')
+def snmp():
+    pass
+
+
+def fetch_mib(mib):
+    import pysnmp_mibs
+    from pysmi.codegen import PySnmpCodeGen
+    from pysmi.compiler import MibCompiler
+    from pysmi.parser import SmiStarParser
+    from pysmi.reader import HttpReader
+    from pysmi.writer import PyFileWriter
+
+    target_directory = os.path.dirname(pysnmp_mibs.__file__)
+
+    reader = HttpReader('mibs.snmplabs.com', 80, '/asn1/@mib@')
+    mibCompiler = MibCompiler(SmiStarParser(), PySnmpCodeGen(), PyFileWriter(target_directory))
+
+    mibCompiler.addSources(reader)
+
+    mibCompiler.compile(mib)
+
+
+@snmp.command(context_settings=CONTEXT_SETTINGS, short_help='Translate MIB name to OIDs in SNMP profiles')
+@click.argument('profile')
```

Comment:
Minor comments:

1.

```suggestion
@click.argument('path')
```

or 

```suggestion
@click.argument('profile_path')
```

When using the command it wasn't clear to me what `profile` mean:
```
§ ddev meta snmp translate-profile -h
Usage: ddev meta snmp translate-profile [OPTIONS] PROFILE

  Do OID translation in a SNMP profile. This isn't a plain replacement, as
  it doesn't preserve comments and indent, but it should automate most of
  the work.

  You'll need to install pysnmp and pysnmp-mibs manually beforehand.

Options:
  -h, --help  Show this message and exit.
```

2. A description for `@click.argument('profile')` can be helpful.